build.py

```diff
@@ -5,6 +5,11 @@
 from leverage import task
 from leverage import path
 
+import hcl2
+import json
```

Comment:
standard libraries goes first, then 3rd party and then locals

https://peps.python.org/pep-0008/#imports

Reply:
nice